content/ways-to-train.md

```diff
@@ -42,208 +39,89 @@ keywords:
-</p>
+If you already have a degree and want to go to university again, you can do a university-led PGCE or PGDE leading to qualified teacher status.
 
-$school-direct-salaried$
+A PGCE or PGDE is an academic qualification that you can use towards masters credits.
 
-You’ll be employed by a school as an unqualified teacher while you work towards your Qualified Teacher Status (QTS).
+You spend about a third of your time in academic learning and the rest in school placements.  
 
-You’ll earn a salary and do not need to pay any tuition fees.
+Courses typically take one year but you may be able to study part time.
 
-This route typically requires 3 years of transferable work history.
+Fees are usually around £9,250 but you can get [funding that you do not have to pay back](/funding-your-training#bursaries-and-scholarships) for certain subjects.
 
-Most school direct programmes lead to a PGCE, but not all do. Check with the training provider before you apply.
+You can also apply for a [student loan](/funding-your-training#tuition-fee-and-maintenance-loans).
 
-<p class="call-to-action__action">
-  <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses">Find a school direct <span>course</span></a>
-</p>
+<a class="button" href="https://www.find-postgraduate-teacher-training.service.gov.uk/">Find university training</a>
 
-$postgraduate-teaching-apprenticeship$
+### Learn on the job in schools near you
 
-You’ll be employed by a school and learn while you’re working.
+If you already have a degree and want to spend as much time as possible in practical learning, you can find a training programme led by a school or group of schools.
 
-You’ll split your time between school and university study, with at least 20% of your time studying.
+You'll only spend about about a fifth of your time in academic learning (for example, written assignments).
 
-<p class="call-to-action__action">
-  <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses">Find a postgraduate teaching <span>apprenticeship</span></a>
-</p>
+Many courses even give you a PGCE or PGDE - an academic qualification which you can use towards masters credits.
 
-$teach-first$
+Courses typically take one year but you may be able to study part time.
 
-The Teach First Leadership Development Programme is a two-year salaried programme.
+Although you often have to pay course fees of around £9,250 you can get [funding that you do not have to pay back](/funding-your-training) for certain subjects.
 
-It combines leadership development with teacher training, and focuses on training exceptional teachers to teach in challenging schools.
+You can also apply for a [student loan](/funding-your-training).
 
-You’ll learn on the job while you work towards Qualified Teacher Status (QTS) and a Postgraduate Diploma in Education (PGDE).
+Some training programmes offer a salary, although these can be harder to find.
```

Comment:
Can we be clear about why they're hard to find - ie is it about your qualifications or experience, or the fact that they are not publicised or online? If there are ones that offer salaries can we link to them or say what the criteria would be?